server/src/models/Settlement.ts

```diff
@@ -1,36 +1,7 @@
 import mongoose, { Document, Schema } from 'mongoose';
 import { TypeServerSettlement } from '../../../SettlementTypes';
 
-// export interface SettlementInterface {
```

Comment:
Delete things instead of leaving commneted code.  You can always find it in git history later.